components/Header/header.js

```diff
@@ -1,72 +1,120 @@
+import Button from "../Button/button";
+import Logo from "../illustrations/logo";
+import Stroke1 from "../illustrations/stroke1";
+import Stroke2 from "../illustrations/stroke2";
+import Step from "../illustrations/stepLike";
+
+const data = [
+  {
+    title: "What is the AsyncAPI Conference?",
+    description:
+      "At the 2022 AsyncAPI Conference, speakers from all around the world will join us online to present how they are using the AsyncAPI Specification in their real-world use cases, how they use tooling to support the specification, and how they have contributed to the project.",
+    button: (
+      <Button
+        text="Visit the AsyncAPI Website"
+        link="https://www.asyncapi.com/"
+        className="text-[21px] py-[14px] px-[26px] mt-[32px]"
+      />
+    ),
+  },
+  {
+    title: "Apply to be a sponsor",
+    description:
+      "We are in the process of looking for those interested in sponsoring the upcoming AsyncAPI Conference.",
+    button: (
+      <Button
+        text="Become a sponsor"
+        link="https://github.com/asyncapi/community/files/9568764/Sponsorship_Prospectus_2022_v2.pdf"
+        className="text-[21px] py-[14px] px-[26px] mt-4 btn-gradient"
+      />
+    ),
+  },
+  {
+    title: "Sign up for updates",
+    description:
+      "Subscribe to email notifications to receive updates on events like: when the official conference schedule is posted, virtual and in-person event details, and more.",
+    button: (
+      <div className="flex h-[56px] mt-[26px] items-center rounded-lg border border-dark-600">
+        <input
+          type="email"
+          className="bg-transparent py-[14px] px-[26px] h-full w-3/4 text-[18px] focus:outline-none indent-3"
+          placeholder="Your email address"
+        />
+        <Button
+          text="Subscribe"
+          className="text-sm h-full w-1/4 border-none rounded-none"
+        />
+      </div>
+    ),
+  },
+];
 
 function Header() {
   return (
-    <div
-      style={{
-        position: "relative",
-        height: "91vh",
-      }}
-    >
-      <div className="absolute">
-        <img src="dots.svg" alt="dots" />
-      </div>
-      <div className="absolute right-0">
-        <Pattern1 className="w-72 sm:w-32" />
+    <div>
+      <div className="mt-10 flex justify-center relative z-50 h-[43vh]">
+        <div className="absolute flex flex-col items-center text-white w-2/4 sm:w-full">
+          <Logo className="" height="112px" width="112px" />
+          <div className="text-center leading-[7rem]">
+            <h1 className="mt-5 text-[100px] sm:text-4xl font-black">
+              AsyncAPI
+            </h1>
+            <h1 className="mt-5 text-[100px] sm:text-4xl -mt-1 font-black">
+              {" "}
+              Conf 2022
+            </h1>
+          </div>
+          <p className="mt-4 text-[30px]">Virtual Tech Conference</p>
+          <p className="mt-1 text-dark-400 text-[28px]">Nov 6-8</p>
```

Comment:
where did you get these dates from 😄 
```suggestion
          <p className="mt-1 text-dark-400 text-[28px]">Nov 3-5</p>
```